web-app/src/pages/Dashboard/container.js

```diff
@@ -27,11 +28,15 @@ const DashboardContainer = Wrapped =>
     }
 
     componentDidMount() {
+      var self = this;
```

Comment:
I don't think should be needed. Or, if it is, let's refactor so it's not lol

Reply:
Yep, forgot to refactor that after removing a promise that wasn't needed